service/post.service.js

```diff
@@ -74,6 +75,34 @@ exports.createComment = async (author, text, postId) => {
   return post;
 };
 
+exports.reactLove = async (author, postId) => {
+  const post = await Post.findOneAndUpdate(
```

Comment:
If we update in this manner, people can spam it. We need to check if the user already love it or not, if already loved, don't update it, if not already loved, update it.

See what can be wrong 😅
![image](https://user-images.githubusercontent.com/15931537/135998631-513e1539-26b7-4a7a-816a-6a4c26bc1471.png)